scripts/main.js

```diff
@@ -9,6 +9,7 @@ var $cancelNewAppFormButton = $('#cancel-create')
 var $appList = $('#app-list')
 var $goBackButton = $('#go-back-btn')
 var $startStopAppButton = $('#main-button')
+var $deletebtn = $('#deletebtn')
```

Comment:
at Hoodie we try to avoid acronyms to make the code more readable. I know most people will likely know what "btn" stands for, but it’s a general rule we try to follow. I’d suggest

``js
$deleteButton = $('#delete-button')
```